.github/workflows/ci.yml

```diff
@@ -123,6 +123,24 @@ jobs:
           globs: |
             **/*.md
 
+      - name: Setup nodejs
```

Comment:
Nit: Is there a Prettier action that could be used either? It would remove the need to take care of yarn and node setup and maintenance.

Reply:
There seems to be more than once, but I'm not sure if that's a good idea. Historically, prettier upgrades would change the output. So it's important to use exactly the same prettier version to format the files and then later to verify on CI that they have been formatted. If we use a dedicated prettier action, then we make it harder to use exactly the same prettier version in all places. Most likely changing the prettier version would require both changes to `package.json` and to the configuration of such github aciton where a specific prettier version could be passed. I'm not saying that's a blocker, it just makes me unsure about choosing such a solution.